subsys/bluetooth/controller/ll_sw/nordic/lll/lll_peripheral_iso.c

```diff
@@ -994,6 +1023,13 @@ static void isr_tx(void *param)
 
 	radio_tmr_hcto_configure_abs(hcto);
 
+#if defined(CONFIG_BT_CTLR_PROFILE_ISR) || \
+	defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
+	radio_tmr_end_capture();
+#endif /* CONFIG_BT_CTLR_PROFILE_ISR ||
+	* HAL_RADIO_GPIO_HAVE_PA_PIN
+	*/
+
```

Comment:
The additional newlines here are not needed

Reply:
It is a single blank line before the next line with `#if` (blank line 1032 between 1031 and 1033), right?

Reply:
I meant that instead of 
```c
#if defined(CONFIG_BT_CTLR_PROFILE_ISR) || \
	defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
	radio_tmr_end_capture();
#endif /* CONFIG_BT_CTLR_PROFILE_ISR ||
	* HAL_RADIO_GPIO_HAVE_PA_PIN
	*/
```
This could simply be

```c
#if defined(CONFIG_BT_CTLR_PROFILE_ISR) || defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
	radio_tmr_end_capture();
#endif /* CONFIG_BT_CTLR_PROFILE_ISR || HAL_RADIO_GPIO_HAVE_PA_PIN */
```